src/Main.java

```diff
@@ -1,6 +1,17 @@
 public class Main {
+
     public static void main(String[] args) {
+        Author lotr = new Author("Джон", " Р.Р.Толкин");
+        Book bookLotr = new Book("Властелин колец", lotr, 1954);
+        bookLotr.setYearPublication(1956);
+        System.out.println("Название книги: " + bookLotr.getNameBook() + " Автор: "
+                + lotr.getName() + lotr.getFamiliya() + " Год публикации:  " + bookLotr.getYearPublication());
```

Comment:
```suggestion
                + bookLotr.getAuthor().getName() + bookLotr.getAuthor().getFamiliya() + " Год публикации:  " + bookLotr.getYearPublication());
```
берём автора у книги, а не с улицы